dataactvalidator/config/sqlrules/d3_detached_award_financial_assistance_1.sql

```diff
@@ -0,0 +1,8 @@
+-- Action type is required for non-aggregate records (i.e., when RecordType = 2)
+SELECT
+    dafa.row_number,
+    dafa.action_type,
+    dafa.record_type
+FROM detached_award_financial_assistance as dafa
+WHERE dafa.record_type = 2
+    AND COALESCE(dafa.action_type, '') = '' ;
```

Comment:
Why this method instead of `AND action_type IS NULL`? If this is faster then that's fine, but I'm just wondering since it seems like this would require a function call and therefore be slower?
Also, just a note, you don't actually have to do `dafa.` if there's only one table in the list. You can just put the column names, just a clean thing.